gvsbuild/projects/__init__.py

```diff
@@ -28,6 +28,7 @@
 from gvsbuild.projects.gperf import Gperf
 from gvsbuild.projects.graphene import Graphene
 from gvsbuild.projects.gsettings_desktop_schemas import GSettingsDesktopSchemas
+from gvsbuild.projects.libsrtp2 import LibSRTP
```

Comment:
Hi @ben-xD, thanks for the contribution! I'll look in to why Ruff isn't enforcing this automatically, but could you please put libsrt2 in alphabetical order with the other imports? Thanks!

Reply:
Hey Dan, sure. I have _leveraged_ my alphabet skills in 0b3fe8227f4202ab3355bb0125869f205dfbe969

Reply:
I noticed a couple more unsorted bits when i ran isort so i fixed them in `34764dff5d65c865fad104c7c4bbe0dfd2126e5b`. I did try to [use ruff](https://stackoverflow.com/a/78156861/7365866) but that didn't do anything:
```
ruff check --select I --fix .
ruff format .
```